src/event.rs

```diff
@@ -232,26 +232,58 @@ where
     reader.read(filter)
 }
 
+/// Mouse capture mode
+#[derive(Debug, Clone, Copy, PartialEq, Eq)]
+pub enum MouseTrackingMode {
+    /// Normal tracking: Send mouse X & Y on button press and release
+    Normal,
+    /// Button-event tracking: Report button motion events (dragging)
+    ButtonEvent,
+    /// Any-event tracking: Report all motion events
+    Any,
+    /// RXVT mouse mode: Allows mouse coordinates of >223
+    RXVTMode,
+    /// SGR mouse mode: Allows mouse coordinates of >223, preferred over RXVT mode
+    SGRMode,
+}
+impl MouseTrackingMode {
+    fn enable(&self) -> &'static str {
```

Comment:
@sigmaSd There's a mistake in the original pull request:
enable and disable are not symmetric. For example Normal should be 1000h and 1000l, but you just copied the reversed sequence from the current EnableMouseCapture.

Can you fix this, because I would like to do some tests where I need to set and unset these modes in sequence.